don't load public key for "myself" because this creates problems with
keys saved in the subjectPublicKeyInfo format (as keys extracted from a
certificate) and is not necessary anyways because the private key already
contains the public key.

diff -u -r1.1 -r1.2
--- tinc/src/net_setup.c	2004/02/19 11:55:20	1.1
+++ tinc/src/net_setup.c	2004/02/19 13:00:43	1.2
@@ -250,8 +250,10 @@
 		return false;
 	}
 
-	if(!read_rsa_public_key(myself->connection))
-		return false;
+	//br1: not needed because private key already contains the public key
+	//br1: removed because it creates problems with subjectPublicKeyInfo format keys
+	//if(!read_rsa_public_key(myself->connection))
+	//	return false;
 
 	if(!get_config_string
 	   (lookup_config(myself->connection->config_tree, "Port"), &myport))